client/src/components/Framework/Framework.jsx

```diff
@@ -37,6 +40,39 @@ export default class Framework extends React.Component {
 		})
 	}
 
+	createPlayers(){
+		let divArr = [];
+		for (let i = 0; i < this.state.scoreboard.length; i++){
+			divArr.push(<div className="sb-player">{this.state.scoreboard[i]}</div>)
+		}
+		return divArr;
+	}
+
+	scoreboardHeader(){
+		let heading;
+		if (this.state.scoreboard.length === 1){
+			heading = '1 Player';
+		} else {
+			heading = `${this.state.scoreboard.length} Players`
+		}
+		return heading;
+	}
+
+	changeScoreboard(){
+		let ranNum = Math.floor(Math.random() * usernames.length)
+		let ranUsers = [];
+
+		if (ranNum === 0){
```

Comment:
This looks _very_ suspect. Whats wrong with `ranNum` being `0`?

Reply:
and if it is bad for it to happen, can't it still happen anyway?

Reply:
I think maybe `Math.floor(Math.random()* usernames.length - 1) + 1` might implement what you want (never being 0)